src/FileEntityInterface.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\file_entity\FileEntityInterface.
+ */
+
+namespace Drupal\file_entity;
+
+/**
+ * File entity interface.
+ */
+interface FileEntityInterface {
+  /**
+   * Get the metadata property value for the given property.
```

Comment:
Comments of getters usually start with "Gets ...". It is also a bit confusing to have "property" in there twice. Usually we leave an empty line after the declaration of class/interface.